assets/rpvoicechat/lang/ru.json

```diff
@@ -58,5 +79,11 @@
     "Gui.ModMenu.denoisingSensitivity.Label": "Распознавание фонового шума",
     "Gui.ModMenu.denoisingSensitivity.Tooltip": "Задаёт чувствительность к фоновому шуму. Аудио, принятое за шум, будет подавляться с максимальной интенсивностью",
     "Gui.ModMenu.denoisingStrength.Label": "Уровень шумоподавления",
-    "Gui.ModMenu.denoisingStrength.Tooltip": "Задаёт интенсивность шумоподавления для звука, распознанного как голос. Уменьшите значение, если ваш голос слишком искажается"
+    "Gui.ModMenu.denoisingStrength.Tooltip": "Задаёт интенсивность шумоподавления для звука, распознанного как голос. Уменьшите значение, если ваш голос слишком искажается",
+    "Gui.ModMenu.toggleChannelGuessing.Label": "Определение аудиоканалов",
+    "Gui.ModMenu.toggleChannelGuessing.Tooltip": "Используем многоканальную запись, если она поддерживается. Пустые аудиоканалы будут выключены для случаем, когда устройство ввода пишет не в стандартные каналы (Например, стереозвук в 3 и 4 каналах вместо 1 и 2). При отключении этой настройки, мод будет использовать все каналы, что, вероятнее всего, уменьшит громкость в 2/4 раза.",
```

Comment:
```suggestion
    "Gui.ModMenu.toggleChannelGuessing.Tooltip": "Мы используем многоканальную запись, если она поддерживается. Пустые аудиоканалы будут выключены для случаев, когда устройство ввода пишет не в стандартные каналы (Например, стереозвук в 3 и 4 каналах вместо 1 и 2). При отключении этой настройки, мод будет использовать все каналы, что, вероятнее всего, уменьшит громкость в 2/4 раза.",
```

Reply:
```suggestion
    "Gui.ModMenu.toggleChannelGuessing.Tooltip": "Использовать многоканальную запись, если она поддерживается. Пустые аудиоканалы будут выключены для случаев, когда устройство ввода пишет не в стандартные каналы (Например, стереозвук в 3 и 4 каналах вместо 1 и 2). При отключении этой настройки, мод будет использовать все каналы, что, вероятнее всего, уменьшит громкость в 2/4 раза.",
```

Reply:
Multichannel audio capture is always used if it is available, unchecking this checkbox will not disable it, it will only disable  automatic discarding of unused audio channels